vizro-core/docs/pages/tutorials/first-dashboard.md

```diff
@@ -37,14 +37,14 @@ Click on the **Run and edit this code in Py.Cafe** link below to live-edit the d
 <!-- vale off -->
 ## Can I break your code?
 <!-- vale on -->
-When you click the link to "Edit live on Py.Cafe" the dashboard is running inside your browser. Any changes you make are local and you don't need to worry about breaking the code for others. Nobody else sees the changes you make unless you save a copy of the project as your own Vizro py.cafe project.
```

Comment:
Hello! Maybe a silly uncertainty, but I'm not 100% sure if py.cafe is a file naming convention or if it's referring to PyCafe and should be updated. I updated it, but can easily revert

Reply:
Hi @LydiaPitts I think your judgement is correct. 👍 This is not talking about the site itself.

Reply:
Great, thanks :)